private/model/api/codegentest/service/awsquerycompatible/awsquerycompatible_test.go

```diff
@@ -0,0 +1,20 @@
+//go:build go1.8 && codegen
+// +build go1.8,codegen
```

Comment:
the `codegen` build tags can be removed here its better that these tests are run without the `codegen` tag to assert SDK's internal protocol deserializer behavior as well.